src/coreComponents/mesh/generators/WellGeneratorBase.cpp

```diff
@@ -528,54 +529,49 @@ void WellGeneratorBase::debugWellGeometry() const
     return;
   }
 
-  std::cout << std::endl;
-  std::cout << "++++++++++++++++++++++++++" << std::endl;
-  std::cout << "WellGeneratorBase = " << getName() << std::endl;
-  std::cout << "MPI rank = " << MpiWrapper::commRank( MPI_COMM_GEOSX ) << std::endl << std::endl;
-  std::cout << "Number of well elements = " << m_numElems << std::endl;
+  std::vector< string > row;
+  std::ostringstream oss;
+  Table table = Table( {
+      Table::ColumnParam{{"Element no."}, Table::Alignment::right},
+      Table::ColumnParam{{"CoordX"}, Table::Alignment::middle},
+      Table::ColumnParam{{"CoordY"}, Table::Alignment::middle},
+      Table::ColumnParam{{"CoordZ"}, Table::Alignment::middle},
+      Table::ColumnParam{{"Prev\nElement"}, Table::Alignment::right},
+      Table::ColumnParam{{"Next\nElement"}, Table::Alignment::right},
+    } );
+
+  string titleName = "InternalWellGenerator " + getName();
+  table.setTitle( titleName );
```

Comment:
```suggestion
  table.setTitle( "InternalWellGenerator " + getName() );
```
or set `titleName` to `const`.